### PR TITLE
docs: update provider setup docs

### PR DESCRIPTION
**Pull request by george-xing**
[Describe your change here]

## Test Plan

[Describe test plan here]

## Deployment instructions

[Add any special deployment instructions here]

### CODE REVIEW

File: docs/docs/providers/hubspot.mdx [hunk 1]
```diff
@@ -71,13 +73,13 @@ import BrowserWindow from '@site/src/components/BrowserWindow';
...
 import TabItem from '@theme/TabItem';
 
-To connect to your customers' HubSpot instances, you'll need to update the redirect URL to point to Supaglue and fetch the API access credentials in your [HubSpot developer account](https://developers.hubspot.com).
+To connect to your customers' HubSpot instances, you'll need to update the redirect URL to point to Supaglue and fetch the API access credentials in your HubSpot developer account.
 
 ### Add Redirect URL to your HubSpot app
 
 Supaglue provides a redirect URL to send information to your app. To add the redirect URL to your HubSpot app:
 
-1. Log in to your HubSpot developer account: <https://app.hubspot.com/developer>
```
Comment (tomkit) [inline docs/docs/providers/hubspot.mdx:80]: Why remove the convenience link?

File: docs/src/components/OAuthScopesSelector/index.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
```
Comment (tomkit) [inline docs/src/components/OAuthScopesSelector/index.tsx:1]: nice

File: docs/docs/providers/hubspot.mdx (hunk 1 above)
Comment (george-xing) [reply docs/docs/providers/hubspot.mdx:80]: The link is broken -> it does an infinite redirect

File: docs/docs/tutorials/custom-fields.md [hunk 3]
```diff
@@ -77,44 +76,30 @@ And return a response that looks like:
...
+
+```shell
+curl -L -X POST 'https://api.supaglue.io/crm/v2/contacts/_upsert' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-H 'x-customer-id: 001' \
+-H 'x-provider-name: hubspot' \
+-H 'x-api-key: <API_KEY_VALUE>' \
+--data-raw '{
+  "record": {
```
Comment (asdfryan) [inline docs/docs/tutorials/custom-fields.md:89]: Note: email is required when creating a new contact

File: docs/docs/providers/hubspot.mdx [hunk 4]
```diff
@@ -71,13 +73,13 @@ import BrowserWindow from '@site/src/components/BrowserWindow';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-To connect to your customers' HubSpot instances, you'll need to update the redirect URL to point to Supaglue and fetch the API access credentials in your [HubSpot developer account](https://developers.hubspot.com).
```
Comment (tomkit) [inline docs/docs/providers/hubspot.mdx:74]: This link still works, right?

**Review by tomkit — APPROVED** [review]
Approving - Ryan left a comment

File: docs/docs/tutorials/custom-fields.md (hunk 3 above)
Comment (george-xing) [reply docs/docs/tutorials/custom-fields.md:97]: fixed

File: docs/docs/providers/hubspot.mdx (hunk 4 above)
Comment (george-xing) [reply docs/docs/providers/hubspot.mdx:74]: added back

**Pull request merged**